modules/core/tests/server/userModel.test.server.js

```diff
@@ -86,6 +89,32 @@ describe('User functionality', function () {
 
   });
 
+  it('should not update the password, when updating the user', function (done) {
```

Comment:
it should not update the password when not updating the password field of the User model